### PR TITLE
Add reviews

### PR DESCRIPTION
**Pull request by NoyaArie**
Add Review entity and calculation for raiting
screen shot 2017-09-18 at 15 07 38

### CODE REVIEW

File: app/assets/stylesheets/restaurants.scss [hunk 1]
```diff
@@ -74,18 +75,18 @@
...
 
           dl {
-            margin: 2px 2px;
+            margin: 10px;
           }
 
-          .list_image {
-            width: 10px;
-            height: 10px;
+          .TenBis_image {
```
Comment (ronylangbord) [inline app/assets/stylesheets/restaurants.scss]: generally, we are naming css classes/object with lower case , so for now: 
`.tesbis_img`

Comment (ronylangbord) [reply app/assets/stylesheets/restaurants.scss]: I recommend reading the style guide in Wiki

File: app/javascript/packs/RestaurantLineItem.jsx [hunk 2]
```diff
@@ -5,13 +5,21 @@ export class RestaurantLineItem extends React.Component {
 
     Accepts10Bis(accepts_10bis) {
         if (accepts_10bis == true) {
-            return <img className="list_image" src={require('../../assets/images/10bis.png')}/> ;
+            return <img className="TenBis_image" src={require('../../assets/images/10bis.png')}/> ;
```
[the remark this answers is not in the document]
Comment (NoyaArie) [reply app/javascript/packs/RestaurantLineItem.jsx]: Do you mean to install react-constant? or: export const TEN_BIS_IMAGE = "...", or just a function variable?

File: app/views/reviews/index.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1 @@
+json.array! @reviews, partial: 'reviews/review', as: :review
```
[the remark this answers is not in the document]
Comment (NoyaArie) [reply app/views/reviews/index.json.jbuilder]: Isn't the scaffold generate it?

**Pull request merged**